doc/sphinxman/source/optking.rst

```diff
@@ -277,30 +276,203 @@ how the geometry of the molecule was input (whether Z-matrix or cartesian, etc.)
 .. code-block:: none
 
    molecule hooh {
-    0 1
-    H  0.850718   0.772960    0.563468
-    O  0.120432   0.684669   -0.035503
-    O -0.120432  -0.684669   -0.035503
-    H -0.850718  -0.772960    0.563468
+     0 1
+     H  0.850718   0.772960    0.563468
+     O  0.120432   0.684669   -0.035503
+     O -0.120432  -0.684669   -0.035503
+     H -0.850718  -0.772960    0.563468
    }
    
    set {
      basis cc-pvdz
      intrafrag_step_limit 0.1
    }
-   
-   dihedrals = [100,110,120,130,140,150]
+
+   lower_bound = [99.99, 109.99, 119.99, 129.99, 149.99]
+   upper_bound = [100, 110, 120, 130, 140, 150]
    PES = []
-   
+
+   for lower, upper in zip(lower_bound, upper_bound):
+   my_string = f"1 2 3 4 {lower} {upper}"
+   set optking ranged_dihedral = $my_string
+   E = optimize('scf')
+   PES.append((upper, E))
+
+   print("\n\tcc-pVDZ SCF energy as a function of phi\n")
+   for point in PES:
+     print("\t%5.1f%20.10f" % (point[0], point[1]))
+
+* To scan the potential energy surface without the |optking__ranged_dihedral| keyword, a zmatrix
+  can be used. **Warning!** rotating dihedrals in large increments without allowing the molecule to relax
+  in between increments can lead to unphysical geometries with overlapping functional groups in larger molecules.
+
+.. code-block:: none
+
+   molecule hooh {
+     0 1 
+     H   
+     O 1 0.95
+     O 2 1.39 1 103 
+     H 3 0.95 2 103 1 D 
+
+     D = 99
+
+     units ang 
+   }
+
+   set {
+     basis cc-pvdz
+     intrafrag_step_limit 0.1 
+     frozen_dihedral (" 1 2 3 4 ")
+   }
+
+   dihedrals = [100, 110, 120, 130, 140, 150]
+   PES = []
+
    for phi in dihedrals:
-     my_string = "1 2 3 4 " + str(phi)
-     set optking fixed_dihedral = $my_string
+     hooh.D = phi 
      E = optimize('scf')
      PES.append((phi, E))
-   
-   print "\n\tcc-pVDZ SCF energy as a function of phi\n"
+
+   print("\n\tcc-pVDZ SCF energy as a function of phi\n")
    for point in PES:
-     print "\t%5.1f%20.10f" % (point[0], point[1])
+     print("\t%5.1f%20.10f" % (point[0], point[1]))
+
+Multi-Fragment Optimizations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _DimerSection:
+
+The intermolecular coordinates are controlled through |optking__interfrag_mode|,
+|optking__frag_ref_atoms|, and |optking__interfrag_coords|.
+
+* Specify the reference points to use for coordinates via |optking__frag_ref_atoms|.
```

Comment:
I don't understand this section at all. What does "reference points to use for coordinates" mean? Am I expected to be able to follow this, or are you assuming the user has already ready some optking-specific documentation?

Reply:
No optking's documentation on this section is actually less detailed. I'll try to clarify that section and add a note that most users should just set `interfrag_mode` to `multi` and let optking handle coordinates.